packages/zambdas/src/ehr/patient-payments/post/index.ts

```
@@ -140,6 +140,10 @@ const performEffect = async (
     };
     const paymentIntent = await stripeClient.paymentIntents.create(paymentIntentInput);
 
+    if (paymentIntent.status !== 'succeeded') {
+      throw new Error(`The card payment was not successful. Try a different card`);
```

Can we throw a defined error type here?